.codecov.yml

```diff
@@ -1,17 +1,17 @@
 # https://docs.codecov.io/docs/codecovyml-reference
 
 # We have
-#   11 * [number of basic OS] + 5 * [number of additional OS]
+#   15 * [number of basic OS] + 2 * [number of additional OS]
```

Comment:
Didn't we want to exclude the additional OS from coverage to get faster coverage reports? Or is this for another PR?

Reply:
Coverage report for `threaded` is turned off, as before - but it still sends a report since it uses the same CI job setup as the other setups

Reply:
Now that we have reorganized everything, couldn't we add an if condition to check for ubuntu, i.e., something like
```yaml
- uses: julia-actions/julia-processcoverage@v1
  if: ${{ matrix.os == 'ubuntu-latest' }}
  with:
    directories: src,examples
- uses: codecov/codecov-action@v1
  if: ${{ matrix.os == 'ubuntu-latest' }}
  with:
    file: ./lcov.info
...
```

Reply:
Doesn't that make the setup just more complicated without gaining anything?

Reply:
Well, we discussed it yesterday that it would be nice to get the coverage report without having to wait for Windows/macOS, since they often take longer to start (and thus to finish). However, if you feel like this is too complicated a change to do as a drive-by in this PR, we can refer this to an issue and resolve this here.

Reply:
Windows and macOS take less than half an hour while other jobs take significantly longer (aiming for 45 minutes, sometimes up to an hour), see https://github.com/trixi-framework/Trixi.jl/actions/runs/1166048768

Reply:
Thus, I would prefer discussing this further in an issue.

Reply:
Yes, it was just a concern since you already noted that the number of parallel macOS jobs are restricted, and we noticed in the past that GH Actions runners weren't available for e.g. Windows during times of high GitHub-wide usage.

Reply:
> Thus, I would prefer discussing this further in an issue.

OK

Reply:
Added to https://github.com/trixi-framework/Trixi.jl/issues/823#issuecomment-905519614

Reply:
Thanks!